community/cypher/cypher/src/test/scala/org/neo4j/cypher/RootPlanAcceptanceTest.scala

```diff
@@ -146,7 +146,8 @@ class RootPlanAcceptanceTest extends ExecutionEngineFunSuite {
   }
 
   test("EstimatedRows should be properly formatted") {
-    given("match n return n").planDescripton.getArguments.get("EstimatedRows") should equal(0)
+    given("match n return n").planDescripton.getArguments.get("EstimatedOperatorCardinality") should equal(0)
+    given("match n return n").planDescripton.getArguments.get("EstimatedProducedRows") should equal(0)
```

Comment:
The column titles feel a bit long. Maybe EstimatedCardinality and EstimatedRows is good enough?

Also I'm missing a documentation section describing this difference to the user, at least briefly.